core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -462,10 +464,12 @@ class ContainerPool(
                 // We grab the size first so we know there has been enough delay for anything we are shutting down
                 val size = toBeRemoved.size()
                 1 to size foreach { _ =>
-                    val ci = toBeRemoved.poll()
-                    if (ci != null) { // should never happen but defensive
+                    val removeJob = toBeRemoved.poll()
+                    if (removeJob != null) {
```

Comment:
I agree. That's why I added the comment to ignore it, if not clearer for @perryibm